Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -256,6 +259,24 @@
             );
 
             launcher = launcher
+                .withColumn(builder =>
+                    builder
+                        .withDataSourceName("ItSystemBusinessTypeId") //Using type id for better search performance and type id is used during sorting (in the parameter mapper)
+                        .withTitle("Forretningstype")
+                        .withId("busitype")
```

Comment:
Er det problematisk med længere Id her? "busitype" kan vel ligeså godt kaldes "businesstype"?

Reply:
det er af hensyn til rækkefølgen af folks eksisterende opsætning. Hvis vi beholder de samme kolonnenavne burde profilen være kompatibel... kan du huske de der `//DONT YOU DARE RENAME`  ?
Det var derfor de er der :-)